input.fs

```diff
@@ -218,8 +218,7 @@ let (|Mouse|Special|Normal|ImeEvent|TextInput|Unrecognized|) (x: InputEvent) =
     |  Key(_, Key.Add)                                            -> Special("kPlus")
     |  Key(_, Key.Subtract)                                       -> Special("kMinus")
     |  Key(_, Key.Divide)                                         -> Special("kDivide")
-    // TODO|  Key(_, Key.Decimal) -> ???
-    // TODO|  Key(_, Key.Separator) -> ???
+    |  Key(_, Key.Separator)                                      -> Special("Bar")
```

Comment:
this one I'm not sure.
see: https://github.com/AvaloniaUI/Avalonia/search?q=Key.Separator

Reply:
according to this thread: https://stackoverflow.com/questions/36666868/uk-keyboard-virtual-key-code-for-the-symbol
`Key.Separator` is the "Enter" key on a numpad and I don't even have a numpad to test it 😂
Maybe map to `<kEnter>`.

Reply:
and maybe `Key.Decimal` to `<kPoint>`?